src/_P012_LCD.ino

```diff
@@ -255,7 +259,7 @@ boolean Plugin_012(uint8_t function, struct EventStruct *event, String& string)
           success = true;
           int colPos  = event->Par2 - 1;
           int rowPos  = event->Par1 - 1;
-          String text = parseStringKeepCase(string, 4);
```

Comment:
Just thinking about this function.
I don't like those true/false arguments when they don't really relate to the function name. (e.g. `setEnabled(bool enable)` is fine)
So maybe better to have some helper function like `parseStringKeepCaseNoTrim`?

Make sure to implement it in the .cpp and .h, so it hardly doesn't add to the build size :)